Documentation/contributor-guide/triage_issues.md

```diff
@@ -4,43 +4,140 @@
+| area/contrib            | Not to be confused with `area/community` this label is specifically used for issues relating to community maintained scripts or files in the `contrib/` directory which aren't part of the core etcd project. |
+| area/documentation      |                                                               |
+| area/tooling            | Generally used in relation to the third party / external utilities or tools that are used in various stages of the etcd build, test or release process, for example tooling to create sboms.          |
+| area/testing            |                                                               |
+| area/robustness-testing |                                                               |
+
+## Step 5 - Prioritise the issue
+
+Placeholder.
+
+## Step 6 - Support new contributors
+
+As part of the `etcd` triage process once the `kind` and `area` have been determined, please consider if the issue would be suitable for a less experienced contributor. The `good first issue` label is a subset of the `help wanted` label, indicating that members have committed to providing extra assistance for new contributors. All `good first issue` items also have the `help wanted` label.
+
+### Help wanted
+
+Items marked with the `help wanted` label need to ensure that they meet these criteria:
+
+* **Low Barrier to Entry** - It should be easy for new contributors.
+
+* **Clear** - The task is agreed upon and does not require further discussions in the community.
+
+* **Goldilocks priority** - The priority should not be so high that a core contributor should do it, but not too low that it isn’t useful enough for a core contributor to spend time reviewing it, answering questions, helping get it into a release, etc.
+
+### Good first issue
+
+Items marked with `good first issue` are intended for first-time contributors. It indicates that members will keep an eye out for these pull requests and shepherd it through our processes.
+
+New contributors should not be left to find an approver, ping for reviews, decipher test commands, or identify that their build failed due to a flake. It is important to make new contributors feel welcome and valued. We should assure them that they will have an extra level of help with their first contribution.
 
-## Validate if an issue is a bug
+After a contributor has successfully completed one or two `good first issue` items, they should be ready to move on to `help wanted` items.
 
-Validate if the issue is indeed a bug. If not, add a comment with findings and close trivial issue. For non-trivial issue, wait to hear back from issue reporter and see if there is any objection. If issue reporter does not reply in 30 days, close the issue. If the problem can not be reproduced or require more information, leave a comment for the issue reporter.
+* **No Barrier to Entry** - The task is something that a new contributor can tackle without advanced setup or domain knowledge.
 
-## Inactive issues
```

Comment:
Even if the issue is not active but if it's not resolved. We can use `stage/tracked` to avoid stale bot close it accidentally. https://github.com/etcd-io/etcd/pull/14108

Reply:
Good suggestion!  Added a section under **Step 7 - Follow up** to cover tracking important issues.